modules/portmaster/src/i18n/helptexts.yaml

```diff
@@ -41,95 +40,92 @@ uiMode:
 pilot-widget-NetworkRating:
   title: Network Rating
   content: |
-    Some features and settings of the Portmaster behave differently, depending on the environment.
+    Control your privacy even when connecting to new networks.
 
-    In order to help the Portmaster provide the most adequate protection, please select "Trusted", "Untrusted" or "Danger" according to the situation you are in.<br>
-    Continue to learn about these different levels.
+    In the Portmaster you configure settings to be active in one environment but not in the other, like allowing sensitive connections at home but not at the public library.
+
+    The only thing you have to do is to change the network rating whenever you connect to a different network.
   nextKey: pilot-widget-NetworkRating-Trusted
 
 pilot-widget-NetworkRating-Trusted:
-  title: Trusted Rating
+  title: "Network Rating: Trusted"
   content: |
-    This rating is for when you expect a network to be secure and help protect you.<br>
-    In most cases, you will use this for your network at home or at friends.
+    You trust the current network to be secure and protect you.
+
+    Examples:
+     - your home network
```

Comment:
I would recommend normal sentence casing and punctuation.

Reply:
what do you mean hear? What would such a recommended sentence be?

Reply:
I saw a use case provided by you (the shield colors) and disagree from a typography perspective. This seems very uncommon and we rarely use it too, think docs homepage or [linux page](https://docs.safing.io/portmaster/install/linux). Remove the dots there to unify this.

Reply:
Ok. I just think it's weird.